xchange-bitfinex/src/main/java/org/knowm/xchange/bitfinex/service/BitfinexTradeServiceRaw.java

```diff
@@ -291,29 +342,40 @@ public BitfinexOfferStatusResponse placeBitfinexFloatingRateLoanOrder(
     String direction = loanOrder.getType() == OrderType.BID ? "loan" : "lend";
 
     BitfinexOfferStatusResponse newOrderResponse =
-        bitfinex.newOffer(
-            apiKey,
-            payloadCreator,
-            signatureCreator,
-            new BitfinexNewOfferRequest(
-                String.valueOf(exchange.getNonceFactory().createValue()),
-                loanOrder.getCurrency(),
-                loanOrder.getOriginalAmount(),
-                new BigDecimal("0.0"),
-                loanOrder.getDayPeriod(),
-                direction));
+        decorateApiCall(
+                () ->
+                    bitfinex.newOffer(
+                        apiKey,
+                        payloadCreator,
+                        signatureCreator,
+                        new BitfinexNewOfferRequest(
+                            String.valueOf(exchange.getNonceFactory().createValue()),
+                            loanOrder.getCurrency(),
+                            loanOrder.getOriginalAmount(),
+                            new BigDecimal("0.0"),
+                            loanOrder.getDayPeriod(),
+                            direction)))
+            .withRetry(retry("trade-floatingRateLoanOrder"))
+            .withRateLimiter(rateLimiter(BITFINEX_RATE_LIMITER))
+            .call();
     return newOrderResponse;
```

Comment:
inline request

Reply:
non-idempotent requests should not have retries on them, you don't want to mistakenly double send an order